packages/components/navbar/src/Navbar.tsx

```diff
@@ -32,6 +32,9 @@ type NavbarOwnProps = CommonProps & {
   /** Navigation displayed on mobile versions */
   mobileNavigation?: React.ReactNode;
 
+  /** breakpoint to determine when to show the mobile navigation */
+  mobileNavigationBp?: 'small' | 'medium';
```

Comment:
What would be the deciding factor to select one or the other? The number of nav items (i.e. the organization settings)?

Reply:
yes, for the organization settings navigation, there is already not enough space for the switcher with the main navigation.

Reply:
We can revisit this logic once we support container queries, we should be able to make it fully dynamic based on the surrounding elements 🤞